graphbolt/src/macro.h

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
this file is placed under `//src`, so it will not be exported as header file on purpose?